docs/reference/esql/functions/percentile.asciidoc

```diff
@@ -1,18 +1,27 @@
 [discrete]
 [[esql-agg-percentile]]
 === `PERCENTILE`
-The value at which a certain percentage of observed values occur. For example,
-the 95th percentile is the value which is greater than 95% of the observed values and
-the 50th percentile is the <<esql-agg-median>>.
 
-[source.merge.styled,esql]
+*Syntax*
+
+[source,esql]
 ----
-include::{esql-specs}/stats_percentile.csv-spec[tag=percentile]
+PERCENTILE(column, percentile)
 ----
-[%header.monospaced.styled,format=dsv,separator=|]
-|===
-include::{esql-specs}/stats_percentile.csv-spec[tag=percentile-result]
-|===
+
+*Parameters*
+
+`column`::
+Column to convert from multiple values to single value.
+
+`percentile`::
+A numeric expression.
```

Comment:
```suggestion
A constant numeric expression.
```
One cannot do e.g. `PERCENTILE(some_field, some_other_field)`, but `PERCENTILE(some_field, 95*(0.5 + 0.25)` is okay.